[Bug] Material Kit PRO React will not install, will not run

### Version
1.9.0
### Reproduction link
https://demos.creative-tim.com/material-kit-pro-react/?_ga=2.126034509.1101420317.1616676140-1212158183.1616676140#/documentation/tutorial
### Operating System
Big Sur 11.2.2
### Device
MacBook Pro
### Browser & Version
Google Chrome Version 89.0.4389.90 (Official Build) (x86_64)
### Steps to reproduce
1. npm install --> throws pages of errors
2. npm run install:clean --> this isn't even a command
3. NODE_PATH=./src --> throws pages of errors
4. npm install --g cross-env --> change start to: "start": "NODE_PATH=./src react-scripts start", --> throws pages of errors
5. npm install --force --> cannot resolve dependencies, throws pages of errors

### What is expected?
I purchased this entire full-stack bundle and it doesn't work. It won't run. I can't actually use this product and likely the entire bundle, and I'm about to dispute the charge on my credit card. The documentation provided isn't helpful or thorough and you're using dependencies that haven't been updated in over a year. I've also tried copying and pasting specific components, manually editing them / css, and installing any dependencies the components need, and I still get the same errors. 
### What is actually happening?
Ironically, when I post my error logs here, it won't let me submit the issue.

---
### Solution

### Additional comments
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: @babel/core@7.9.0
npm ERR! node_modules/@babel/core
npm ERR!   @babel/core@"7.9.0" from react-scripts@3.4.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@"3.4.1" from the root project
npm ERR!   @babel/core@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!   node_modules/@svgr/webpack
npm ERR!     @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!     node_modules/react-scripts
npm ERR!       react-scripts@"3.4.1" from the root project
npm ERR!   9 more (babel-jest, babel-loader, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @babel/core@"^7.13.0" from @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12
npm ERR! node_modules/@babel/preset-env/node_modules/@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining
npm ERR!   @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@"^7.13.12" from @babel/preset-env@7.13.12
npm ERR!   node_modules/@babel/preset-env
npm ERR!     @babel/preset-env@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!     node_modules/@svgr/webpack
npm ERR!       @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!       node_modules/react-scripts
npm ERR!
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR!
npm ERR! See /Users/jessicagallagher/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/jessicagallagher/.npm/_logs/2021-03-27T12_28_38_502Z-debug.log
➜  material-kit-pro-react-v1.9.0 git:(main) ✗ npm install --g cross-env

added 7 packages, and audited 8 packages in 1s

found 0 vulnerabilities
➜  material-kit-pro-react-v1.9.0 git:(main) ✗ npm install
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR!
npm ERR! Found: @babel/core@7.9.0
npm ERR! node_modules/@babel/core
npm ERR!   @babel/core@"7.9.0" from react-scripts@3.4.1
npm ERR!   node_modules/react-scripts
npm ERR!     react-scripts@"3.4.1" from the root project
npm ERR!   @babel/core@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!   node_modules/@svgr/webpack
npm ERR!     @svgr/webpack@"4.3.3" from react-scripts@3.4.1
npm ERR!     node_modules/react-scripts
npm ERR!       react-scripts@"3.4.1" from the root project
npm ERR!   9 more (babel-jest, babel-loader, ...)
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer @babel/core@"^7.13.0" from @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@7.13.12
npm ERR! node_modules/@babel/preset-env/node_modules/@babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining
npm ERR!   @babel/plugin-bugfix-v8-spread-parameters-in-optional-chaining@"^7.13.12" from @babel/preset-env@7.13.12
npm ERR!   node_modules/@babel/preset-env
npm ERR!     @babel/preset-env@"^7.4.5" from @svgr/webpack@4.3.3
npm ERR!   
<!-- generated by creative-tim-issues. DO NOT REMOVE -->

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -1,5 +1,58 @@
 # Change Log
 
+## [1.10.0] 2021-05-12
+### Bug fixing
+- https://github.com/creativetimofficial/ct-material-kit-pro-react/issues/82
+- https://github.com/creativetimofficial/ct-material-kit-pro-react/issues/78
+- https://github.com/creativetimofficial/ct-material-kit-pro-react/issues/76
+- https://github.com/creativetimofficial/ct-material-kit-pro-react/issues/69
+- https://github.com/creativetimofficial/ct-material-kit-pro-react/issues/49
+### Major style changes
+
+### Deleted components
+
+### Added components
++ `@babel/core@7.14.0` (to stop some of the console warnings)
+### Deleted dependencies
+- history (We'll use `BrowserRouter` instead of `Router` and `history`)
+- `react-google-maps` (We'll use simple Google Maps API)
+- `@types/googlemaps`
+- `@types/markerclustererplus`
+- `react-swipeable-views` (no longer maintained)
+### Added dependencies
+
+### Updated dependencies
+```
+@material-ui/core         4.10.1   →   4.11.4
+@material-ui/icons         4.9.1   →   4.11.2
+animate.css                4.1.0   →    4.1.1
+classnames                 2.2.6   →    2.3.1
+moment                    2.26.0   →   2.29.1
+node-sass                 4.14.1   →    6.0.0
+nouislider                14.5.0   →   15.1.0
+react                    16.13.1   →   17.0.2
+react-datetime            2.16.3   →    3.0.4
+react-dom                16.13.1   →   17.0.2
+react-image-gallery        1.0.7   →    1.0.9
+react-scripts              3.4.1   →    4.0.3
+react-slick               0.26.1   →   0.28.1
+gulp-append-prepend        1.0.8   →    1.0.9
+eslint-config-prettier    6.11.0   →    8.3.0
+eslint-plugin-prettier     3.1.3   →    3.4.0
+eslint-plugin-react       7.20.0   →   7.23.2
+prettier                   2.0.5   →    2.3.0
+typescript                 3.9.3   →    4.2.4
+```
+### Warning
+_Warnings might appear while doing an npm install - they do not affect the UI or the functionality of the product, and they appear because of NodeJS and not from the product itself._
+```
+npm WARN react-animate-on-scroll@2.1.5 requires a peer of react@>= 15.4.1 < 17.0.0-0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-datetime@3.0.4 requires a peer of react@^16.5.0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-tagsinput@3.19.0 requires a peer of react@^16.0.0 || ^15.0.0 || ^0.14.0 but none is installed. You must install peer dependencies yourself.
+npm WARN react-swipeable@5.5.1 requires a peer of react@^16.0.0-0 but none is installed. You must install peer dependencies yourself.
+```
+_You will also have the following message: found 80 vulnerabilities (1 low, 79 moderate). This comes from react-scripts, and will be fixed in the next version. NOTE: the product works as expected with these vulnerabilities._
+
 ## [1.9.0] 2020-06-05
 ### Bug fixing
 - Changed the usage of `react-image-gallery` to new API, check `src/views/ProductPage/ProductPage.js`

--- a/README.md
+++ b/README.md
@@ -1,7 +1,7 @@
 # [Material Kit PRO React](https://demos.creative-tim.com/material-kit-pro-react) [![Tweet](https://img.shields.io/twitter/url/http/shields.io.svg?style=social&logo=twitter)](https://twitter.com/home?status=Material%20Kit%20PRO%20is%20a%20Bootstrap%20UI%20Kit%20with%20a%20fresh,%20new%20design%20inspired%20by%20Google's%20Material%20Design%20%E2%9D%A4%EF%B8%8Fhttps%3A//demos.creative-tim.com/material-kit-pro/presentation.html%20%23bootstrap%20%23material%20%23design%20%23uikit%20%23premium%20%20via%20%40CreativeTim)
 
 
- ![version](https://img.shields.io/badge/version-1.9.0-blue.svg)  [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/ct-material-kit-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-kit-pro-react/issues?q=is%3Aopen+is%3Aissue) [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/ct-material-kit-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-kit-pro-react/issues?q=is%3Aissue+is%3Aclosed) [![Join the chat at https://gitter.im/NIT-dgp/General](https://badges.gitter.im/NIT-dgp/General.svg)](https://gitter.im/creative-tim-general/Lobby) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
+ ![version](https://img.shields.io/badge/version-1.10.0-blue.svg)  [![GitHub issues open](https://img.shields.io/github/issues/creativetimofficial/ct-material-kit-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-kit-pro-react/issues?q=is%3Aopen+is%3Aissue) [![GitHub issues closed](https://img.shields.io/github/issues-closed-raw/creativetimofficial/ct-material-kit-pro-react.svg?maxAge=2592000)](https://github.com/creativetimofficial/ct-material-kit-pro-react/issues?q=is%3Aissue+is%3Aclosed) [![Join the chat at https://gitter.im/NIT-dgp/General](https://badges.gitter.im/NIT-dgp/General.svg)](https://gitter.im/creative-tim-general/Lobby) [![Chat](https://img.shields.io/badge/chat-on%20discord-7289da.svg)](https://discord.gg/E4aHAQy)
 
 ![Product Presentation Image](https://s3.amazonaws.com/creativetim_bucket/products/89/original/opt_mkp_react_thumbnail.jpg)
 
@@ -312,7 +312,7 @@ If you have questions or need help integrating the product please [contact us](h
 
 ## Licensing
 
-- Copyright 2020 Creative Tim (https://www.creative-tim.com/)
+- Copyright 2021 Creative Tim (https://www.creative-tim.com/)
 
 - Creative Tim [license](https://www.creative-tim.com/license)
 

